balboa-agent/src/main/java/com/socrata/balboa/agent/MetricConsumer.java

```diff
@@ -106,7 +117,6 @@ private int processNamespace(File dir) {
                 // Flush the file to JMS after every read
 
                 // TODO I don't think we need this so commenting out the below line.
```

Comment:
This actually surfaces an issue with the PR.  Because we removed the explicit call to flush the write buffer when the Metric Consumer process is done.  it was a little mysterious because that call was kind of thrown in that method.  I am deleting the comments but need to make a change to make sure all the elements get flushed from the internal buffer without exposing internal representation.  Stand by.